core/sawtooth/cli/transaction.py

```diff
@@ -155,26 +155,12 @@ def print_trans_info(args, web_client, tsctids):
         "Identifier": "TRANS",
         "InBlock": "BLOCK",
         "Status": "STATUS",
-        "TransactionType": "TXNTYPE",
-        "Update": {
```

Comment:
Sorry, the above comment doesn't answer your question. Not related, but we need to print different transaction information due to different transaction type, for example,  the update of ValidatorTransaction is different from the update of the other transaction type.